examples/src/queries/task.ts

```diff
@@ -56,6 +62,55 @@ export async function queryTasks(queryParams: QueryParams) {
   return response;
 }
 
+// Provide the query parameters, all fields are optional
+// const args: QueryParams = {
+//   id: taskid!,
+//   md5: null,
+//   user_address: null,
+//   tasktype: null,
+//   taskstatus: null,
+// };
+
+export async function queryConciseTasks(queryParams: QueryParams) {
+  const response: PaginationResult<ConciseTask[]> = await ServiceHelper.loadTaskList(
+    queryParams
+  );
+  const task: ConciseTask = response.data[0];
+  console.log("Task ID", task._id);
+  console.log("Task user address", task.user_address);
+  console.log("Task md5", task.md5);
+  console.log("Task status", task.status);
+  console.log("Task type", task.task_type);
+  console.log("Task submit time", task.submit_time);
+  console.log("Task start time", task.process_started);
+  console.log("Task finish time", task.process_finished);
+  console.log("Task auto submit status", task.auto_submit_status);
+  console.log("Task proof submit mode", task.proof_submit_mode);
+
+  return response;
+}
+
+export async function queryConciseThenDetail(queryParams: QueryParams) {
+  // Query concise task 
+  const concise = await queryConciseTasks(queryParams);
+
+  // Get the id of the task
+  const id = concise.data[0]._id as string;
+  const queryById: QueryParams = {
+    id: id,
+    md5: null,
+    user_address: null,
+    tasktype: null,
+    taskstatus: null,
+  };
+
+
+  // Query task detail with task id
+  const taskDetail = await queryTasks(queryById);
```

Comment:
why we still use queryTasks?
First this is example internal function. Also this function is using loadTasks which we do not recommand user to use.

Example is to show developer how to use our helper lib. Thus we better do not use any example only function.

Thus, in fact I think queryConciseTasks() should be something like queryTaskList(), which just show how to use loadTaskList().

For queryOneTaskIdThenDetail, we should use loadTaskList first to get the id.
Then use the new getTaskDetailFromId() to get the task detail.

Also add a queryMultiTaskIdsThenDetail() which use loadTaskList to get some ids, then use getTasksDetailFromIds() to get result.